vendordeps/photonlib.json

```diff
@@ -0,0 +1,41 @@
+{
+    "fileName": "photonlib.json",
+    "name": "photonlib",
+    "version": "v2023.2.1",
```

Comment:
FYI, latest version is v2023.3.0

Reply:
Cool, we can update code and limelight tomorrow.
Release notes since v2023.2.1: [v2023.2.2](https://github.com/PhotonVision/photonvision/releases/tag/v2023.2.2), [v2023.3.0](https://github.com/PhotonVision/photonvision/releases/tag/v2023.3.0)

Reply:
updated it